### ISSUE TITLE
BUG - SEV 3 - All - Non-compliant copy on Notifications screen

### ISSUE THREAD

**Issue by mistymg** [department-of-veterans-affairs/va-mobile-app#7393]
## What happened?

Copy on Notifications screen doesn't follow VA style guide principles.

## Specs:

- Device: iPhone SE
- OS: iOS
- User Account: Demo mode
- Accessibility State: n/a

## Steps to Reproduce

- Sign in using demo mode
- Tap Profile on Home screen
- Tap Settings
- Tap Notifications

## Desired behavior

Copy on Notifications screen should comply with VA style guide (i.e., capitalization, contractions, more helpful labels).

## Acceptance Criteria

Changes to following copy:

- [ ] Personalize Notifications header now reads: Personalized notifications
- [ ] Body copy now reads: Select which notifications you'd like to receive.
- [ ] Helper text now reads: Your privacy is important to us. We won't display any personal information in your notifications.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
- Impact: Low 
- Frequency: Low 
- 3 - Low

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [X] Steps to reproduce are defined
- [X] Desired behavior is added
- [X] Labels added (front-end, back-end, relevant feature, accessibility, etc)
- [X] Estimate of 1 added (for front-end tickets)
- [X] Added either to the relevant feature epic (if found during new feature implementation), or the [bug epic](https://app.zenhub.com/workspaces/va-mobile-product-view-610035bc5395bb000e62e529/issues/gh/department-of-veterans-affairs/va-mobile-app/5367)

**Comment by DJUltraTom** [comment]
@rbontrager will this change require a corresponding detox ticket?
@mistymg is this ticket designed to only be for demo mode or was the intention for a change to copy on the actual app?

**Comment by mistymg** [comment]
@DJUltraTom This isn't just for demo mode. This is a change that needs to be made in production for all users.

**Comment by DJUltraTom** [comment]
Sending back to "in progress" as the changes are not appearing outside of demo mode.
<img width="369" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeVJVQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a3569d4a28dd7fcd621736670ea5f2a528b2a5be/Screenshot_20231212-130603.png" alt="Screenshot_20231212-130603.png" />


<img width="600" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeHhVQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--49dadcd7b68412194c53597f3c823d203c5a0127/IMG_BC2192ECA966-1.jpeg" alt="IMG_BC2192ECA966-1.jpeg" />

**Comment by liztownd** [comment]
@timwright12 @DJUltraTom @mistymg - I've dug in a bit and these 2 items:

-  Appointment Reminders toggle copy now reads: Upcoming appointments
-  Secure Message Alerts toggle copy now reads: New messages

Are coming from the API VEText endpoint (`/v0/push/prefs/{endpointSid}` in the mobile api), this is not something we set in the frontend of the app. Is this something I should dig into further on the API side? Or is it a backend issue and there should be a backend ticket for it? 

And if we're hitting the right endpoint, shouldn't what the mobile API returns match what's being returned for the website as well? A BE engineer might be able to answer if they know if the endpoint has changed, I suppose... 

(Side Note: I went looking on staging.va.gov and I can't find secure messaging there to see what the current verbiage for these 2 things are on the website.)

**Comment by dumathane** [comment]
@DJUltraTom I talked to Misty and we are going to remove those two AC's that were still failing and Misty is going to make a spin off ticket for them due to their complexity and additional requirements. Can you revisit this ticket without those AC's and let me know if we are good to merge now?

**Comment by DJUltraTom** [comment]
@dumathane  Happy to revisit this, I cant find the new ticket to address that AC, can you or @mistymg  drop a link to it in the comments here so we have a paper trail?

**Comment by mistymg** [comment]
Hey @DJUltraTom @dumathane, I was going to make a Content spin-off ticket to dig deeper into those 2 ACs we removed.  Not an Eng spin-off ticket. There's coordination with another team needed before handing them back off to Eng again.

**Comment by DJUltraTom** [comment]
given the above comments This ticket now meets the AC.
QA approves this ticket

**Comment by mistymg** [comment]
@DJUltraTom @liztownd @dumathane Circling back to this closed ticket. If you remember, we decided not to change the copy in the toggles. When I sign in to Judy Morrison test user, everything looked good. But in demo mode, the toggle copy was changed. First SS below is test user (correct) and second SS is demo mode (incorrect). Can we reopen this ticket and fix this?
![unnamed](https://github.com/department-of-veterans-affairs/va-mobile-app/assets/102106810/07a543de-e83b-474d-b448-dce00e8e6e74)
![Screenshot_20240209-131128_VA](https://github.com/department-of-veterans-affairs/va-mobile-app/assets/102106810/0c89ddaa-9dda-4f34-b59b-553cbf587168)
